test/test_utils/security.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+
+from enum import IntEnum
+
+from test import test_utils
+
+
+class CVESeverity(IntEnum):
+    UNDEFINED = 0
+    INFORMATIONAL = 1
+    LOW = 2
+    MEDIUM = 3
+    HIGH = 4
+    CRITICAL = 5
+
+
+class ScanVulnerabilityList:
```

Comment:
is there a place to use your equivalence function from test_utils here?

Reply:
I initially added it because I was planning on comparing the Allowlist directly with the ECR Scan Vulnerabilities List.

But the subtraction function is helpful in cases where there are vulnerabilities that are now fixed, and the allowlist is just old. Using `__sub__` instead of `__cmp__` helps us differentiate between when there are unknown vulnerabilities vs when there are outdated vulnerabilities on allowlist.

But the `__cmp__` is useful for further development that @shantanutrip might need, which is why I've left it in.